extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache

```diff
@@ -37,7 +37,7 @@
 			window._sp_.bootstrap('{{domain}}/api/v1/ARecoveryEngine/delivery');
 
 			window.addEventListener('load', function () {
-				require(['ext.wikia.aRecoveryEngine.recovery.sourcePointHelper'], function (helper) {
+				require(['ext.wikia.aRecoveryEngine.recovery.sourcePoint'], function (helper) {
```

Comment:
What will happen if a page will be cached and ask for module which doesn't exist anymore?